go install not working / metrics package not found

**Description**

I tried to install the tool with `go install github.com/docker/hub-tool`.
It's currently failing, because the package `github.com/docker/compose-cli/metrics` cannot be found.
I checked the metrics package. It seems that they reorganized the structure in the main branch.

**Steps to reproduce the issue:**
1. `go install github.com/docker/hub-tool`

**Describe the results you received:**

```
go/src/github.com/docker/hub-tool/internal/metrics/metrics.go:22:2: cannot find package "github.com/docker/compose-cli/metrics" in any of:
	/usr/lib/go/src/github.com/docker/compose-cli/metrics (from $GOROOT)
	/home/cmuench/go/src/github.com/docker/compose-cli/metrics (from $GOPATH)
```

**Describe the results you expected:**

Tool can be installed without an error.


**Output of `go version`:**
go version go1.15.7 linux/amd64

--- a/README.md
+++ b/README.md
@@ -25,7 +25,7 @@ perform Hub maintenance tasks.
 
 OR
 
-- Install from sources: `go install github.com/docker/hub-tool`
+- Install from sources: `GO111MODULE=on go get github.com/docker/hub-tool`
 
 ### Login to Docker Hub
 

